README.md

```diff
@@ -2030,23 +2102,21 @@ In case anyone is from **Lviv**, let's keep in touch
 
 ---
 
-   /\     /\
-  {  `---'  }
+/\ /\
+ { `---'  }
   {  O   O  }
   ~~>  V  <~~
    \  \|/  /
-    `-----'__
```

Comment:
probably this was not the intention of you commit pls check the lines you modified